data/json/items/comestibles/meat_dishes.json

```
@@ -223,6 +223,43 @@
     "description": "Lean sausage made from salt-cured offal, that has been cooked.",
     "flags": [ "EATEN_HOT" ]
   },
+  {
+    "id": "sausage_blood",
+    "copy-from": "sausage_raw",
```

to get "better results" try to copy from sausage instead.

```suggestion
    "copy-from": "sausage",
```